code/modules/power/supermatter/supermatter.dm

```diff
@@ -428,21 +432,27 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		bzcomp += clamp(max(removed.gases[/datum/gas/bz][MOLES]/combined_gas, 0) - bzcomp, -1, gas_change_rate)
 		n2ocomp += clamp(max(removed.gases[/datum/gas/nitrous_oxide][MOLES]/combined_gas, 0) - n2ocomp, -1, gas_change_rate)
 		n2comp += clamp(max(removed.gases[/datum/gas/nitrogen][MOLES]/combined_gas, 0) - n2comp, -1, gas_change_rate)
+		freoncomp += clamp(max(removed.gases[/datum/gas/freon][MOLES]/combined_gas, 0) - freoncomp, -1, gas_change_rate)
 
 		//We're concerned about pluoxium being too easy to abuse at low percents, so we make sure there's a substantial amount.
 		if(pluoxiumcomp >= 0.15)
 			pluoxiumbonus = 1    //makes pluoxium only work at 15%+
 		else
 			pluoxiumbonus = 0
 
+		if(freoncomp <= 0.03)
+			freonbonus = 1
+		else
+			freonbonus = 0.01 //stop any meaningful power generation
```

Comment:
is this a full stop? or do you want to keep the minuscule rad output as a sign. Is there a visible difference between this and freonbonus = 0?

Reply:
i've made the bonus to 0 instead of 0.01. This way will effectively stop any and all form of radiation output from the crystal